src/systemctl/systemctl.c

```diff
@@ -4282,7 +4290,29 @@ static void print_status_info(
         else
                 printf("\n");
 
-        if (endswith(i->id, ".timer")) {
+        is_timer = endswith(i->id, ".timer");
+
+        if (!is_timer && !strv_isempty(i->triggered_by)) {
+                char **trigger;
+                bool found = false;
+                UnitActiveState state;
+                off = ansi_normal();
+
+                STRV_FOREACH(trigger, i->triggered_by) {
+                        r = get_state_one_unit(bus, *trigger, &state);
+                        if (r < 0) {
+                                continue;
+                        }
```

Comment:
no {} around single-line if blocks. As mentioned further down, i'd just ignore the return code here though